### PR TITLE
feat: disable sv calling in tnscope

### PR DESCRIPTION
**Pull request by fevac**
#### Changed

- Disabled SV calling in TNscope

## Documentation

- [ ] N/A
- [ ] Updated Balsamic documentation to reflect the changes as needed for this PR.
    - [Document Name]

### Feature Tests

- [ ] N/A
- [ ] Test [Description]
    - [Screenshot]

### Pipeline Integrity Tests

- **Report deliver (generation of the `.hk` file)**
    - [x] N/A
    - [ ] Verified
- **TGA T/O Workflow**
    - [ ] N/A
    - [x] Verified
- **TGA T/N Workflow**
    - [ ] N/A
    - [x] Verified
- **UMI T/O Workflow**
    - [ ] N/A
    - [x] Verified
- **UMI T/N Workflow**
    - [ ] N/A
    - [x] Verified
- **WGS T/O Workflow**
    - [ ] N/A
    - [x] Verified
- **WGS T/N Workflow**
    - [ ] N/A
    - [x] Verified
- **QC Workflow**
    - [x] N/A
    - [ ] Verified
- **PON Workflow**
    - [x] N/A
    - [ ] Verified

### Documentation

- **Atlas documentation**
    - [x] N/A
    - [ ] Updated: [Link]
- **Web portal for Clinical Genomics**
    - [x] N/A
    - [ ] Updated: [Link]

### Panel of Normal specific criteria

- [ ] The PR includes the addition of a new Panel of Normals
- [ ] The samples have been verified to adhere to the sample selection criteria on [Atlas PoN creation instructions for Balsamic](https://atlas.scilifelab.se/infrastructure/BALSAMIC/balsamic_pon/#procedure)

### User Changes

- [x] N/A
- [ ] This PR affects the output files or results.
    - [ ] User feedback is considered unnecessary because [Justification].
    - [ ] Affected users have been included in the development process and given a chance to provide feedback.

### Infrastructure Changes

- **Stored files in Housekeeper**
    - [x] N/A
    - [ ] Updated: [Link]
- **CG (CLI and delivered/uploaded files)**
    - [x] N/A
    - [ ] Updated: [Link]
- **Servers (configuration files on Hasta)**
    - [x] N/A
    - [ ] Updated: [Link]
- **Scout interface**
    - [x] N/A
    - [ ] Updated: [Link]

### Validation criteria

Validation criteria to be added to validation report PR: [LINK-TO-VALIDATION-REPORT-PR from the [validations repository](https://github.com/Clinical-Genomics/validations/)]

**Version specific criteria**

- Text here or N/A

> [!IMPORTANT]
> One of the below checkboxes for validation need to be checked 

- [ ] Added version specific validation criteria to validation report
- [ ] Changes validated in standard sections: [validation-section]
- [ ] Validation criteria not necessary

## Checklist

> [!IMPORTANT]  
> Ensure that all checkboxes below are ticked before merging.

### For Developers

- **PR Description**
    - [ ] Provided a comprehensive description of the PR.
    - [ ] Linked relevant user stories or issues to the PR.
- **Documentation**
    - [ ] Verified and updated documentation if necessary.
- **Validation criteria**
    - [ ] Completed the validation criteria section of the template.
- **Tests**
    - [ ] Described and tested the functionality addressed in the PR.
    - [ ] Ensured integration of the new code with existing workflows.
    - [ ] Confirmed that meaningful unit tests were added for the changes introduced.
    - [ ] Checked that the PR has successfully passed all relevant code smells and coverage checks.
- **Review**
    - [ ] Addressed and resolved all the feedback provided during the code review process.
    - [ ] Obtained final approval from designated reviewers.

### For Reviewers

- **Code**
    - [ ] Code implements the intended features or fixes the reported issue.
    - [ ] Code follows the project's coding standards and style guide.
- **Documentation**
    - [ ] Pipeline changes are well-documented in the CHANGELOG and relevant documentation.
- **Validation criteria**
    - [ ] The author has completed the validation criteria section of the template
- **Tests**
    - [ ] The author provided a description of their manual testing, including consideration of edge cases and boundary
      conditions where applicable, with satisfactory results.
- **Review**
    - [ ] Confirmed that the developer has addressed all the comments during the code review.

### CODE REVIEW

**Comment by fevac** [comment]
On fleetjay TNscope without SV calling uses less than half the amount of memory.
```
- SNV+SV: 
s          h:m:s    max_rss   max_vms   max_uss   max_pss   io_in  io_out  mean_load  cpu_time
2105.0470  0:35:05  21434.90  24029.06  21416.26  21424.04  0.00   0.00    2836.07    59701.18


- SNV only:
s          h:m:s    max_rss  max_vms  max_uss  max_pss  io_in  io_out  mean_load  cpu_time
1777.9047  0:29:37  7562.47  9069.88  7543.03  7550.92  0.00   0.00    3052.55    54271.81
```

**Comment by fevac** [comment]
Still seeing unrelated issues with balsamic not being properly tracked

File: tests/models/test_params_models.py [hunk 1]
```diff
@@ -215,7 +215,7 @@ def test_umiparams_consensuscall():
 
 
 def test_umiparams_tnscope():
```
Comment (mathiasbio) [inline tests/models/test_params_models.py:217]:
```suggestion
def test_params_tnscope():
```

**Review by mathiasbio — APPROVED** [review]
Super well done, love the refactoring 🌟

**Comment by fevac** [comment]
Develop branch when run via cg fails collecting the metrics due to this rule post_process_tnscope_tga failing and so tests here also fail to fully complete (this rule doesn't fail when running out of cg though), however they pass the tnscope part. So merging.
